cloudnet-modules/cloudnet-cloudperms/src/main/java/de/dytanic/cloudnet/ext/cloudperms/gomint/GoMintCloudNetCloudPermissionsPlugin.java

```diff
@@ -0,0 +1,54 @@
+package de.dytanic.cloudnet.ext.cloudperms.gomint;
+
+import de.dytanic.cloudnet.driver.CloudNetDriver;
+import de.dytanic.cloudnet.driver.permission.CachedPermissionManagement;
+import de.dytanic.cloudnet.ext.cloudperms.CloudPermissionsManagement;
+import de.dytanic.cloudnet.ext.cloudperms.gomint.listener.GoMintCloudNetCloudPermissionsPlayerListener;
+import de.dytanic.cloudnet.wrapper.Wrapper;
+import io.gomint.GoMint;
+import io.gomint.entity.EntityPlayer;
+import io.gomint.plugin.Plugin;
+import io.gomint.plugin.PluginName;
+import io.gomint.plugin.Version;
+
+@PluginName("CloudNet-CloudPerms")
+@Version(major = 1, minor = 2)
+public final class GoMintCloudNetCloudPermissionsPlugin extends Plugin {
+
+    private static GoMintCloudNetCloudPermissionsPlugin instance;
+
+    public static GoMintCloudNetCloudPermissionsPlugin getInstance() {
+        return GoMintCloudNetCloudPermissionsPlugin.instance;
+    }
+
+    private CachedPermissionManagement permissionsManagement;
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=6226234)